js/main.js

```diff
@@ -1,9 +1,44 @@
-const table = new Table()
+class BlackJack {
```

Comment:
Gribētos iebilsts pret šādu risinājumu. main.js failā nevajadzētu veidot klases. Ja ir nepieciešamība, tad veidojam jaunu failu ar attiecīgo klasi. Šajā gadījumā:
1. Izveidojam failut blackJack.js
2. Importējam to iekš index.html
3. Darbojamies jaunizveidotajā failā.

Reply:
Ok, tā arī saprtu, ka main.js fails ir tikai kā galvenās klases inicializācijas vieta, lai startētu programmu javaskriptā un citās objektorientētās progrmmās